daemon/tests/harness/mod.rs

```diff
@@ -352,6 +352,15 @@ impl Taker {
             .unwrap()
             .unwrap();
     }
+
+    pub async fn force_close(&self, order_id: OrderId) {
+        self.system
+            .cfd_actor_addr
+            .send(taker_cfd::Commit { order_id })
+            .await
+            .unwrap()
+            .unwrap();
+    }
```

Comment:
Doesn't need to happen now but I thought I'd point you towards https://github.com/itchysats/itchysats/issues/774.

If we would have these functions directly on the actor system, we could likely re-design the tests to leverage composition even more and directly operate on the `TakerActorSystem` hence not needing to write this indirection.

Reply:
that's interesting! I'm happy to give it a shot on top of this. if not the full redesign, even a short macro `deliver_command!` would get rid of all these functions ;D

Reply:
#865 for example highlights that this is useful outside of the tests as well. I think the tests here just show us a shortcoming of the abstraction in our production code.